src/index.native.animated.js

```diff
@@ -135,8 +151,19 @@ class SwipeableViews extends Component {
     }
   }
 
+  setIndexWithoutTransition(newIndex) {
```

Comment:
We have two complementary options:
 - provide a `animateTransitions` property as we have for the `index.native.scroll.js` version
 - provide a `springConfig` property as we have for the `browser` version